src/client/_clientwindow.py

```diff
@@ -647,66 +637,69 @@ def cleanup(self):
         """
         self.state = ClientState.SHUTDOWN
 
-        self.progress.setWindowTitle("FAF is shutting down")
-        self.progress.setMinimum(0)
-        self.progress.setMaximum(0)
-        self.progress.setValue(0)
-        self.progress.setCancelButton(None)
-        self.progress.show()
+        progress = QtWidgets.QProgressDialog()
+        progress.setMinimum(0)
```

Comment:
Yeah, once should be enough. We can make a commit to fix that with  'git commit --fixup <original commit sha>', and we'll meld it in once we start rebasing the branch.